src/core/Provider.ts

```diff
@@ -697,7 +697,9 @@ export const Provider: React.FC<{
       // XXX this is not very efficient
       mKeys(state).forEach((a) => {
         const aState = mGet(state, a) as AtomState<unknown>
-        freeze(aState.value)
```

Comment:
If CodeSandbox CI can be package for latest PR commit that would be awesome.

Reply:
I think it did?

Reply:
Sorry, my mistake.  CodeSandbox edited original comment. Smarter than I fought.

Reply:
You can also click the green ✔️ mark left to the commit.